backend/parseport/settings.py

```diff
@@ -18,103 +18,92 @@
 # See https://docs.djangoproject.com/en/3.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = 'kxreeb3bds$oibo7ex#f3bi5r+d(1x5zljo-#ms=i2%ih-!pvn'
+SECRET_KEY = os.getenv('DJANGO_SECRET_KEY', 'django-insecure-1234567890')
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
-
-ALLOWED_HOSTS = [
-    'localhost',
-    'http://localhost:4200'
-]
-
+DEBUG = os.getenv('DJANGO_DEBUG', True) == 1
```

Comment:
This is quite confusing and I also think it's not doing what you intended it to do:

1. I think it's generally safer to have DEBUG off by default
2. If I set FOO=1 then `os.getenv('FOO')` returns the string `"1"` and not 1
3. `True == 1` but `"1" != 1` and on prod it's actually behaving ok because `"0" != 1`

My suggestion:
`DEBUG = int(os.getenv('DJANGO_DEBUG', 0)) == 1`

Reply:
Oh, my bad! Good spot.